.changeset/fifty-guests-rescue.md

```diff
@@ -0,0 +1,5 @@
+---
+"@latticexyz/store-sync": patch
+---
+
+Added numerical array column types to the decoded indexer instead of JSON blobs.
```

Comment:
```suggestion
- Added numerical array column types to the decoded indexer instead of JSON blobs
- Bumped the Postgres column size for `int32`, `uint32`, `int64`, and `uint64` types to avoid overflows
```